Makefile

```diff
@@ -20,7 +23,9 @@ test-ci:
 	@NODE_ENV=test NODE_PATH=lib XUNIT_FILE=$(CIRCLE_TEST_REPORTS)/mocha.xml \
 	./node_modules/.bin/istanbul cover \
 	./node_modules/mocha/bin/_mocha --report lcovonly -- -R xunit-file
+ifdef CODECLIMATE_REPO_TOKEN
```

Comment:
`.gitignore` changes reverted. Makefile `ifdef` around codeclimate token kept due to CircleCI failures complaining about it missing when CircleCI builds the forked-branch of this PR.